dds_web/static/swagger.yaml

```
@@ -467,6 +467,379 @@ paths:
+                    example:
+                      [
+                        Name,
+                        Contact Email,
+                        Days In Available,
+                        Days In Expired,
+                        External Display Name,
+                        Public ID,
+                        Safespring Endpoint,
+                        Size,
+                      ]
+  /motd:
+    post:
+      tags:
+        - superadmin
+      summary: Add a Message of the Day.
+      operationId: addMOTD
+      parameters:
+        - $ref: "#/components/parameters/defaultHeader"
+      responses:
+        "401":
+          $ref: "#/components/responses/UnauthorizedToken"
+        "400":
+          $ref: "#/components/responses/BadRequest"
+        "500":
+          $ref: "#/components/responses/InternalServerlError"
+        "200":
+          description: succesful operation
+          content:
+            application/json:
+              schema:
+                type: string
+                example: "{message: The MOTD was added to the database.}"
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                message:
+                  type: string
+                  example: "This is a message of the day"
+    get:
+      tags:
+        - superadmin
+      summary: Return list of all active MOTDs to super admin.
+      operationId: allUnits
+      parameters:
+        - $ref: "#/components/parameters/defaultHeader"
+      responses:
+        "401":
+          $ref: "#/components/responses/UnauthorizedToken"
+        "400":
+          $ref: "#/components/responses/BadRequest"
+        "500":
+          $ref: "#/components/responses/InternalServerlError"
+        "200":
+          description: succesful operation
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  motds:
+                    type: array
+                    items:
+                      $ref: "#/components/schemas/MotdInfo"
+                  keys:
+                    type: array
+                    items:
+                      type: string
+                    example: [MOTD ID, Message, Created]
+    put:
+      tags:
+        - superadmin
+      summary: Deactivate MOTD.
+      operationId: deactivateMOTD
+      parameters:
+        - $ref: "#/components/parameters/defaultHeader"
+      responses:
+        "401":
+          $ref: "#/components/responses/UnauthorizedToken"
+        "400":
+          $ref: "#/components/responses/BadRequest"
+        "500":
+          $ref: "#/components/responses/InternalServerlError"
+        "200":
+          description: succesful operation
+          content:
+            application/json:
+              schema:
+                type: string
+                example: "{message: The MOTD was deactivated in the database.}"
```

```suggestion
                example: "{message: The MOTD was successfully deactivated in the database.}"
```
I guess it is not super important that the messages are exact, but it is good to have them like that for consistency sake